rust/datafusion/src/logical_plan/mod.rs

```diff
@@ -890,11 +955,16 @@ pub enum LogicalPlan {
         /// The output schema of the explain (2 columns of text)
         schema: Box<Schema>,
     },
+    /// Extension operator defined outside of DataFusion
+    Extension {
+        /// The runtime extension operator
+        node: Arc<dyn ExtensionPlanNode>,
+    },
 }
 
 impl LogicalPlan {
     /// Get a reference to the logical plan's schema
-    pub fn schema(&self) -> &Box<Schema> {
+    pub fn schema(&self) -> &Schema {
```

Comment:
Returning a `&Box<>` requires implementers to store their `Schema` objects in a `Box` -- using a `&Schema` provides additional flexibility